src/BinaryBuilderBase.jl

```diff
@@ -70,9 +70,19 @@ const allow_ecryptfs = Ref(false)
 const use_ccache = Ref(false)
 const bootstrap_list = Symbol[]
 
+const _artifacts_toml = @path Pkg.Artifacts.find_artifacts_toml(@__FILE__)
+
 function get_bbb_version(dir=@__DIR__, uuid="7f725544-6523-48cd-82d1-3fa08ff4056e")
     # Get BinaryBuilder.jl's version and git sha
-    version = Pkg.TOML.parsefile(joinpath(dir, "..", "Project.toml"))["version"]
+    version = Pkg.activate(".") do
```

Comment:
Note that you're ignoring `dir`.  This breaks https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/11af33aadd1d502bfd2c90a7c8382e2550439580/src/BinaryBuilder.jl#L53-L54